analytics/cdk/emr-eks/README.md

```diff
@@ -0,0 +1,14 @@
+# Welcome to your CDK TypeScript project
```

Comment:
There are no instructions on configuring or deploying the pattern. It would be good to explain the entrypoint in `bin/emr-eks.ts`, how to add more teams, possibly customizing context values etc. Another set of instructions to run `npm install` and deploying the CDK app will also be useful.